praktikum/poglavlja/12_dinamicka_alokacija/index.rst

```diff
@@ -0,0 +1,104 @@
+Dinamička alokacija memorije
+============================
+
+Podeli svoje utiske o poglavlju na anketi https://docs.google.com/forms/d/1xdwvmDBf-dHSrliFcBA9KL1egNNWD9tYxEjdQRzcqSg
```

Comment:
Koristiti anketu: https://docs.google.com/forms/d/1v87L2JkRzXnvglKgZtM-rPJjTuzB-93z9LHasXQMbA0

I postaviti kao link "ovde" (imaš primer u prethodnim poglavljima kako se to radi, promaklo mi je samo kod Relje da to ispravimo).